gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineTestingIntegrationTest.groovy

```diff
@@ -68,62 +81,98 @@ class BaselineTestingIntegrationTest extends IntegrationSpec {
         import net.jqwik.api.Property;
         import net.jqwik.api.ForAll;
         
-        class TestClass6 { 
+        class JqwikTest { 
             @Property
             void test(@ForAll byte value) {}
         }
         '''.stripIndent(true)
 
-    def '#gradleVersionNumber: capable of running both junit4 and junit5 tests'() {
+    def '#gradleVersionNumber: runs JUnit4 tests'() {
         when:
         gradleVersion = gradleVersionNumber
```

Comment:
Only one of these tests was testing on both Gradle 7 and 8. I've updated these tests so we're testing both Gradle versions on most of the tests (all the tests where it may be relevant).